.github/azure-steps.yml

```diff
@@ -99,10 +99,15 @@ steps:
 #    condition: eq(variables['python_version'], '3.8')
 
   - script: |
+      cat requirements.txt
       python -m pip install -U -r requirements.txt
+      python -m pip uninstall -y spacy-legacy
+      python -m pip install "git+https://github.com/polm/spacy-legacy.git@feature/entity-linker-component-v1"
```

Comment:
I'm not sure why this is necessary to get it working, but it should be removed before merging anyway. It will need to wait on https://github.com/explosion/spacy-legacy/pull/31